sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 3.1.1 (Unreleased)
+## 3.1.1 (2021-06-08)
```

Comment:
just triple checking in case I missed the PR, did you update the package version per se already?

Reply:
It was updated automatically after the last release:

https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_version.py#L7

Reply:
ohh smart! 
In .NET it assumes the next release is a minor one. so I have by default `3.2.0-beta.1` haha. 
ok all good here